sklearn/metrics/cluster/_supervised.py

```diff
@@ -622,7 +622,8 @@ def mutual_info_score(labels_true, labels_pred, *, contingency=None):
     Returns
     -------
     mi : float
-       Mutual information, a non-negative value
+       Mutual information, a non-negative value, measured in nats using the
+       natural logarithm
```

Comment:
```suggestion
       natural logarithm.
```

Reply:
OK do you want me to take out the (U) and (V) clarifications? Thanks for your review!

Reply:
I don't have strong opinion on that. Let's wait for another reviewer.

Reply:
Gotcha. Sounds good. Thanks!